newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/UserSurveyQuestion.js

```diff
@@ -112,6 +112,7 @@ const styles = {
     borderTopLeftRadius: 8,
     borderTopRightRadius: 8,
     width: '100%',
+    aspectRatio: '292 / 103',
```

Comment:
wow that's specific 😅

Reply:
I did not choose that 😅 And since 103 is a prime number, I cannot simplify the ratio